test/shared/src/main/scala/zio/test/Gen.scala

```diff
@@ -86,7 +86,7 @@ final case class Gen[-R, +A](sample: ZStream[R, Nothing, Sample[R, A]]) { self =
    * meet the specified predicate.
    */
   def filterNot(f: A => Boolean): Gen[R, A] =
-    filter(a => !(f(a)))
+    filter(a => !f(a))
```

Comment:
Stray change?

Reply:
Minor refactoring.
Shouldn't I commit this change in this PR?

Reply:
No need to add additional parentheses to code that is already compiling.

Reply:
Yeah, I removed unnecessary parentheses.

Reply:
👍